openverse_catalog/dags/providers/provider_api_scripts/brooklyn_museum.py

```diff
@@ -40,7 +40,10 @@ def main():
     while condition:
         query_param = _get_query_param(offset=offset)
         objects_batch = _get_object_json(query_param=query_param)
-        logger.debug(len(objects_batch))
+        if objects_batch is None:
+            logger.debug("No data")
+        else:
+            logger.debug(len(objects_batch))
```

Comment:
Since we're touching this, perhaps we could make this more descriptive?
```suggestion
        logger.debug(f"Number of batches: {len(objects_batch or [])}")
```